doc/api/http.md

```diff
@@ -2498,7 +2498,7 @@ added: v0.9.12
 
 * `msesc` {number}
 * `callback` {Function} Optional function to be called when a timeout
-occurs, Same as binding to the `timeout` event.
+occurs. Same as binding to the `timeout` event.
```

Comment:
```suggestion
  occurs. Same as binding to the `timeout` event.
```

Reply:
Thanks. I'll look into adding https://github.com/freeletics/remark-lint-blank-lines to lint for these kinds of things. I also wonder if I need to review https://github.com/nodejs/node/pull/40042 a little closer.

Reply:
Actually, seems like I was right the first time. Making the suggested change but not concerned about #40042 as much.

Reply:
It looks like using `remark` to format the markdown (similar to using `prettier` for JS code) would have corrected this. I wonder if we should do that for all the files, and then have either a Makefile task or a GitHub Action to keep things that way.

Reply:
_(FWIW `prettier` also works for markdown documents, it's not limited to JS code.)_

Aren't we using `remark` already? That's what the name of the repo suggests: `nodejs/remark-preset-lint-node`.

Reply:
We use `remark` to lint but not to format. I'm in the midst of changing that.